flexget/plugins/urlrewrite_newpct.py

```diff
@@ -8,11 +8,20 @@
 from flexget.utils.requests import Session
 from flexget.utils.soup import get_soup
 
+from flexget.utils.search import normalize_unicode
+
+from requests.auth import AuthBase
+
 log = logging.getLogger('newpct')
 
-requests = Session()
-requests.headers.update({'User-Agent': 'Mozilla/4.0 (compatible; MSIE 5.5; Windows NT)'})
-requests.set_domain_delay('imdb.com', '2 seconds')
+
+class NewPCTAuth(AuthBase):
+    """Attaches HTTP Authentication to the given Request object."""
```

Comment:
Can we fix this docstring to say what this method actually does? This isn't http authentication, this is user-agent spoofing.